pkg/admissioncontroller/webhooks/auth/seed/graph/eventhandler_seed.go

```diff
@@ -129,6 +128,13 @@ func (g *graph) handleSeedDelete(seed *gardencorev1beta1.Seed) {
 	g.deleteVertex(VertexTypeSeed, "", seed.Name)
 }
 
+func (g *graph) handleManagedSeedIfSeedBelongsToIt(ctx context.Context, seedName string) {
+	// error is ignored here since we cannot do anything meaningful with it
+	if managedSeed, err := kutil.GetManagedSeedByName(ctx, g.client, seedName); err != nil && managedSeed != nil {
```

Comment:
```suggestion
	if managedSeed, err := kutil.GetManagedSeedByName(ctx, g.client, seedName); err == nil && managedSeed != nil {
```
Or simply `managedSeed != nil` since if err is non-nil, managedSeed will always be nil.

Reply:
Thanks...